README.md

```diff
@@ -4,14 +4,19 @@
 [![Downloads][downloads-image]][downloads-url]
 [![Build Status][build-image]][repo-url]
 
-[feathers-lowdb](repo-url) is a database service adapter for [Lowdb][lowdb-repo], a small JSON database for Node, Electron and the browser powered by Lodash. LowDB can store data in-memory or on the filesystem which makes it useful as a persistent storage without a separate database server.
+[feathers-lowdb](repo-url) is a database service adapter for [Lowdb][lowdb-repo], a small JSON database for Node, Electron and the browser powered by Lodash. LowDB can store data in-memory or on the filesystem which makes it useful as a persistent storage without a separate database server. **Only works with `lowdb@^1.0.0` for now**
```

Comment:
Maybe say `only tested with`.

The wrapper should be version agnostic so theoretically if your build system supports esm then you should be able to use higher versions of lowdb

Reply:
I tested it extensively and tried to upgrade. A lot changed. Lodash is gone, we have to await more things, we have to call `await db.read()`, chaining is gone. See my updated issue #34 

It can't physically work, we depend on lodash features like castArray and extend, which are completely absent from V3. We could check if db._ exists and throw with a helpful compatibility message if it doesn't.

I think we could get lots of Feathers users on board if we simply bundled lowDB and offered YAML output by default. We don't have to depend on YAML to do that with ES Modules. Lots of projects are dropping CommonJs support, like LowDB did, so we could too. I've seen even server oriented projects do it.

LowDB V2+ are a completely different database... @typicode put a lot of time into it, changed the licensing, started a 100 sponsor fund raiser and then released V3 with MIT licensing.

Reply:
I see. Thanks for the detailed testing

Reply:
I will publish the release in a hour or so